osf/migrations/0045_remove_invalid_social_entries.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-07-28 13:17
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from osf.models import OSFUser as osfuser
+
+import logging
+
+logger = logging.getLogger(__file__)
+
+def remove_invalid_social_entries(state, *args, **kwargs):
+    OSFUser = state.get_model('osf', 'osfuser')
+    targets = OSFUser.objects.filter()
```

Comment:
In order to avoid iterating through every user, you can just iterate through the ones that have anything in `social`:

```python
 OSFUser.objects.exclude(social={})
```

On prod, this is ~7,600 vs ~75,000.